test/data/ideal-gas.yaml

```diff
@@ -0,0 +1,142 @@
+units: {length: cm, time: s, quantity: mol, energy: cal}
+
+phases:
+- name: simple
+  thermo: ideal-gas
+  elements: [N, O]
+  species: [O2, NO, N2]
+  state: {T: 500.0, P: 10 atm, X: {O2: 0.21, N2: 0.79}}
+
+- name: duplicate-species
+  thermo: ideal-gas
+  elements: [N, O]
+  species: [O2, NO, N2, NO]
+  state: {T: 300.0, P: 1 atm, X: {O2: 0.21, N2: 0.79}}
+
+- name: element-override
+  thermo: ideal-gas
+  elements: [N, O, Ar]
+  species: [O2, NO, N2, AR]
+  note: replace Argon with custom element
+  state: {T: 900.0, P: 5 atm, Y: {O2: 0.4, N2: 0.4, AR: 0.2}}
+
+- name: element-remote
+  thermo: ideal-gas
+  elements:
+  - default: [N, O]
+  - test_subdir/species-elements.yaml/isotopes: [Ar]
```

Comment:
Are these files relative to the CWD of the interpreter/program, the Cantera data directory, or the location of the input file? (If this is documented and I missed it, sorry about that 😄)

Reply:
This searches the Cantera data path, which starts with the CWD unless you go out of your way to change it. I thought I had documented this, but apparently not (or at least, it's not in the YAML format API docs).

Reply:
OK, the docs have been updated, after I reminded myself that this searches the directory containing the current input file first, then the Cantera data path.